ros2_socketcan_msgs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2_socketcan_msgs</name>
+  <version>1.1.0</version>
+  <description>Messages for SocketCAN</description>
+  <maintainer email="josh@electrifiedautonomy.com">Josh Whitley</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+
+  <build_depend>rosidl_default_generators</build_depend>
+
+  <depend>std_msgs</depend>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
+
+  <exec_depend>rosidl_default_runtime</exec_depend>
+
+  <member_of_group>rosidl_interface_packages</member_of_group>
+
+  <export>
+    <build_type>ament_cmake</build_type>
+  </export>
+</package>
```

Comment:
@JWhitleyWork How about splitting the PR that adds `ros2_socketcan_msgs` into a separate PR? Seeing the PR title, it seems you didn't intend to add this package.

Reply:
Done. Once this one is merged, I'll request the one to add the _msgs package.